EasyPost.Tests/BatchTest.cs

```diff
@@ -1,118 +1,220 @@
-﻿using EasyPost;
-
-using System;
+﻿using System.Collections.Generic;
 using System.Linq;
-using System.Collections.Generic;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace EasyPost.Tests {
+namespace EasyPost.Tests
+{
     [TestClass]
-    public class BatchTest {
-        Dictionary<string, object> fromAddress;
-        Dictionary<string, object> toAddress;
-        Dictionary<string, object> shipmentParameters;
-        Dictionary<string, object> batchShipmentParameters;
+    public class BatchTest
+    {
+        private Dictionary<string, object> batchShipmentParameters;
+
+        private Dictionary<string, object> fromAddress;
+
+        private Dictionary<string, object> shipmentParameters;
+
+        private Dictionary<string, object> toAddress;
+
+        public Batch CreateBatch()
+        {
+            Dictionary<string, object> parameters = new Dictionary<string, object>
+            {
+                {
+                    "reference", "EasyPostCSharpTest"
+                },
+                {
+                    "shipments", new List<Dictionary<string, object>>
+                    {
+                        batchShipmentParameters
+                    }
+                }
+            };
+
+            return Batch.Create(parameters);
+        }
 
         [TestInitialize]
-        public void Initialize() {
+        public void Initialize()
+        {
             ClientManager.SetCurrent("NvBX2hFF44SVvTPtYjF0zQ");
 
-            fromAddress = new Dictionary<string, object>() {
-                { "name", "Andrew Tribone" },
-                { "street1", "480 Fell St" },
-                { "street2", "#3" },
-                { "city", "San Francisco" },
-                { "state", "CA" },
-                { "country", "US" },
-                { "zip", "94102" }
+            fromAddress = new Dictionary<string, object>
+            {
+                {
+                    "name", "Andrew Tribone"
+                },
+                {
+                    "street1", "480 Fell St"
+                },
+                {
+                    "street2", "#3"
+                },
+                {
+                    "city", "San Francisco"
+                },
+                {
+                    "state", "CA"
+                },
+                {
+                    "country", "US"
+                },
+                {
+                    "zip", "94102"
+                }
             };
```

Comment:
Is this convention to break these out to their own lines or personal choice? Like the class properties, this seems to hinder readability for single items because it spaces it out so far you lose context on what these belonged to in the beginning. 7 lines vs 20 reads much nicer to me. Thoughts?

Also, are we not able to just have one dictionary? One set of opening/closing brackets containing all the keys for the address?

This comment is meant to encompass every Dictionary that is split up this way, whatever we determine should be applied to each one.

Reply:
The linter automatically rearranged the code that way, so there's some setting in our .editorconfig related to this. Trying to figure out which one.

As for your second comment, this is how you initialize a dictionary in C#. Think of it as a list of key-value pairs. The JSON-ish way you're probably imagining doesn't work in C#.

Reply:
I believe I found the correct setting, but it seems it would affect a lot of initializers throughout the code (plus, despite me changing it, the IDE is still resetting the layout to what it is now, so it might not be the right setting after all).

Reply:
Hmm. I'd really rather not have them so spread out if we can avoid it (because they are their own separate single-item dictionary entries)

Reply:
I'll keep trying to change the settings.

Reply:
After talking this over, our whole library uses dictionaries of dictionaries which is...bad. Unfortunately this linting here is done correctly, it's just ugly because of how the library is setup. In an ideal world, we'd just allow plain dictionaries through and not nested ones. That though is a totally separate task that we can tackle elsewhere.